### PR TITLE
[17.0][MIG] purchase_invoice_method: Migration to 17.0

### PR DESCRIPTION
**Pull request by vvrossem**

### CODE REVIEW

File: purchase_invoice_method/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Purchase Invoice Method",
     "summary": """
         Allow to force the invoice method of a purchase""",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
```
Comment (yankinmax) [inline purchase_invoice_method/__manifest__.py:8]: the easiest migration ever!! cool, thanks

**Review by yankinmax — APPROVED** [review]

**Review by traitrantrobz — APPROVED** [review]
LGTM 👍

**Comment by traitrantrobz** [comment]
@OCA-git-bot migration purchase_invoice_method. Please add this to the [Migration to version 17.0](https://github.com/OCA/purchase-workflow/issues/2087)

**Pull request merged**